socs/Lakeshore/Lakeshore372.py

```diff
@@ -1,10 +1,14 @@
 # Lakeshore372.py
 
+import logging
 import socket
 import sys
+import time
 
 import numpy as np
 
+logging.basicConfig(level=logging.INFO, format='%(levelname)s - %(message)s')
```

Comment:
This is a good addition I think. However, now we'll have two logging systems, `txaio`'s logging in the agent, and this standard logging module in the driver. This probably makes sense, since I wouldn't want to require `txaio` here. However, we should handle setting the log level similarly so they can be set simultaneously.

This is done in the agent via the `LOGLEVEL` environment variable:

https://github.com/simonsobs/socs/blob/702af44481d1cdbca34e3c217b73e51ff59fdb8f/socs/agents/lakeshore372/agent.py#L1112

Can we set it the same way here? You'll need to map the strings used to set the txaio logging to the appropriate `logging.LEVEL` from the `logging` module. `txaio` follows the log levels from twisted: https://docs.twistedmatrix.com/en/stable/core/howto/logger.html#log-levels

And here are the log levels from `logging`: https://docs.python.org/3/library/logging.html#logging-levels

Reply:
I'm not sure I understand exactly what the request is here. You're saying you want to make sure that the `logging` level `INFO` means the same type of logging that comes from `txaio`'s `info`? And if it doesn't, change it accordingly? In perusing those docs, it wasn't clear to me what the right `logging` level should be—I guess `debug`? (`info` says the arguments for `logging.info` are interpreted the same as `debug` which confuses me because it makes me think `info` is fine).

Reply:
The log levels in `txaio` are "debug, info, warn, error, critical" while the log levels in `logging` are "DEBUG, INFO, WARNING, ERROR, CRITICAL". My comment was in regards to 'warn' and 'WARNING' being different.

The documentation for `logging` specifies these numerical values and specifically mentions this is particularly useful if you need to define your own levels to override the defaults. This could be useful to make the log level names uniform between the two systems.

Reply:
> (`info` says the arguments for `logging.info` are interpreted the same as `debug` which confuses me because it makes me think `info` is fine).

That's just talking about the arguments to the function `logging.info()`. The docstring for `logging.debug()` has a long explanation of the `kwargs`. To save repetition they point you there, as the args at the same in `info()`.